### DISCUSSION TITLE
Add Danish language

### DISCUSSION THREAD

**Discussion by roaldarbol** [ente-io/ente#3500] [Enhancements]
Hey guys! First off, congrats on the Mozilla funding!

I’d love to see all apps translated to Danish and would be happy to help put with it. Can you add it to the CrowdIn? :-)

**Comment by mnvr** [comment]
Added! Thank you 🙏

**Comment by roaldarbol** [comment]
Reached 100% for Auth!

**Reply by roaldarbol** [reply]
I accidentally changed a translation and can’t go back to the correct one. It’s “next” which should be “næste”, in case you’re able to change it.

**Reply by mnvr** [reply]
Awesome! The Danish option should appear in the next release.

(I've changed "næste" too)
